src/nvim/eval.c

```diff
@@ -15577,6 +15577,53 @@ static void f_split(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   p_cpo = save_cpo;
 }
 
+/// "stdpath()" helper for list results
+void stdpath_list_helper(const XDGVarType xdg, typval_T *rettv)
```

Comment:
`helper` is not descriptive. `get_xdg_var_list` perhaps (can’t really use `stdpath[s]` prefix because this is not in `stdpaths.c`)? For a static function no prefix is fine (it should be static BTW).

Also need `FUNC_ATTR_NONNULL_ALL`.